tasks/rewriteSourceMaps.js

```diff
@@ -0,0 +1,15 @@
+/* jshint node:true */
+
+module.exports = function (grunt) {
```

Comment:
For the `dist` task, yes. The `dist` task has source maps generated with inline source. When they  are written, they point to a URL in the `src` directory which is really weird in dev tools (ex. `node_modules/src/Promise.ts`). This task tells the source map that the URL should just be pointing to the directory the source map is in (ex. `node_modules/dojo-core/_debug/Promise.ts`).